deny.toml

```diff
@@ -13,34 +13,53 @@ ignore = [
 # when upgrading.
 # Please keep this list alphabetically sorted.
 skip = [
+     { name = "base16ct", version = "0.1.1" },
      { name = "base64", version = "<0.21" },
      { name = "block-buffer", version = "<0.10" },
-     { name = "clap", version = "3.2.23" },
      { name = "clap_lex", version = "0.2.4" },
+     { name = "clap_lex", version = "0.2.4" },
+     { name = "clap", version = "3.2.23" },
+     { name = "clap", version = "3.2.23" },
      { name = "convert_case", version = "0.4.0" },
-     { name = "darling", version = "<0.14" },
+     { name = "convert_case", version = "0.4.0" },
+     { name = "curve25519-dalek", version = "3.2.0" },
```

Comment:
what still pulls this in? this shouldn't be in the dep tree anymore

Reply:
oh lol, okay iroh will upgrade, but we still need some deps to be released first, so I guess we have to accept the duplicate deps for the moment.